diff --git a/ZalithLauncher/src/main/java/com/movtery/zalithlauncher/game/account/AccountsManager.kt b/ZalithLauncher/src/main/java/com/movtery/zalithlauncher/game/account/AccountsManager.kt
index e7b0b750..06c10263 100644
--- a/ZalithLauncher/src/main/java/com/movtery/zalithlauncher/game/account/AccountsManager.kt
+++ b/ZalithLauncher/src/main/java/com/movtery/zalithlauncher/game/account/AccountsManager.kt
@@ -28,7 +28,6 @@ import com.movtery.zalithlauncher.game.account.auth_server.data.AuthServerDao
 import com.movtery.zalithlauncher.path.PathManager
 import com.movtery.zalithlauncher.setting.AllSettings
 import com.movtery.zalithlauncher.ui.androidText
-import com.movtery.zalithlauncher.utils.isInGreaterChina
 import com.movtery.zalithlauncher.utils.logging.Logger
 import com.movtery.zalithlauncher.utils.network.isNetworkAvailable
 import kotlinx.coroutines.CoroutineScope
@@ -213,17 +212,8 @@ object AccountsManager {
      */
     private fun refreshCurrentAccountState() {
         val currentAccount = getCurrentAccount()
-        val isOffline = checkLimit()
-        _currentAccountFlow.update {
-            //若处于非正版状态，不允许使用账号
-            if (isOffline) null else currentAccount
-        }
-        _isOffline.update { isOffline }
-    }
-
-    private fun checkLimit(): Boolean {
-        val circumventLimit = File(PathManager.DIR_FILES_EXTERNAL, "circumventLimit")
-        return !circumventLimit.exists() && !isInGreaterChina() && !hasMicrosoftAccount()
+        _currentAccountFlow.update { currentAccount }
+        _isOffline.update { false }
     }
 
     /**
