diff --git a/ZalithLauncher/src/main/java/com/movtery/zalithlauncher/path/UrlManager.kt b/ZalithLauncher/src/main/java/com/movtery/zalithlauncher/path/UrlManager.kt
index 7b2b2408..c99145d3 100644
--- a/ZalithLauncher/src/main/java/com/movtery/zalithlauncher/path/UrlManager.kt
+++ b/ZalithLauncher/src/main/java/com/movtery/zalithlauncher/path/UrlManager.kt
@@ -46,8 +46,8 @@ const val URL_MCMOD: String = "https://www.mcmod.cn/"
 const val URL_MINECRAFT_VERSION_REPOS: String = "https://piston-meta.mojang.com/mc/game/version_manifest_v2.json"
 const val URL_MINECRAFT_ASSETS_INDEX: String = "https://launchermeta.mojang.com/v1/packages"
 const val URL_MINECRAFT_PURCHASE = "https://www.xbox.com/games/store/minecraft-java-bedrock-edition-for-pc/9nxp44l49shj"
-const val URL_PROJECT: String = "https://github.com/ZalithLauncher/ZalithLauncher2"
-const val URL_PROJECT_INFO: String = "https://api.github.com/repos/ZalithLauncher/Zalith-Info/contents/v2"
+const val URL_PROJECT: String = "https://github.com/Astreele/ZalithLauncher2-patches"
+const val URL_PROJECT_INFO: String = "https://api.github.com/repos/Astreele/ZalithLauncher2-patches/contents"
 const val URL_COMMUNITY: String = "https://github.com/ZalithLauncher/ZalithLauncher2/graphs/contributors"
 const val URL_WEBLATE: String = "https://hosted.weblate.org/projects/zalithlauncher2"

